CONTRIBUTING.md

```diff
@@ -0,0 +1,37 @@
+# Contributing to LiveChat Platform Docs
+
+Thank you for taking the time to improve the [LiveChat](https://www.livechat.com/) documentation 🙌🏻. Hopefully, the following guidelines will help you do it in the most efficient way.
+
+## Find your case
+
+- [I have a problem with the LiveChat App](#i-have-a-problem-with-the-livechat-app)
+- [I'm building an app for the Marketplace and I got stuck](#im-building-an-app-for-the-marketplace-and-i-got-stuck)
+- [I found a bug/typo in your documentation](#i-found-a-bugtypo-in-your-documentation)
+- [I have an improvement suggestion for the docs](#i-have-an-improvement-suggestion-for-the-docs)
+
+## I have a problem with the LiveChat App
+
+If you've run into problems with our LiveChat Application (setting it up, trobule with configuration, not receiving chats), [contact our 24/7 support](https://direct.lc.chat/1520/125), who specialize in solving such cases.
+
+Please, don't create an issue in this repository.
+
+## I'm building an app for the Marketplace and I got stuck
+
+We've been there 😅. If you're stuck in the development process, constantly getting an error from our APIs, contact us at developers@livechat.com or [hop on Discord](https://discord.gg/rFbJkYQFwp).
+
+## I found a bug/typo in your documentation
```

Comment:
@jakubsikora, please pay attention to this instruction if it's okay :) we don't want to confuse people :P